doc/whats_new/v0.21.rst

```diff
@@ -553,3 +553,8 @@ These changes mostly affect library developers.
   when `fit` is called twice with the same data, the ouput of
   `predict`, `predict_proba`, `transform`, and `decision_function` does not
   change. :issue:`12328` by :user:`Nicolas Hug <NicolasHug>`
+
+- Add estimators tags: these are annotations of estimators that allow
```

Comment:
Maybe this belongs in "Multiple modules" and the comment here should be that some checks will now be run selectively depending on the estimator's tags